src/app/core/services/issue.service.ts

```diff
@@ -104,11 +104,19 @@ export class IssueService {
   }
 
   getIssue(id: number): Observable<Issue> {
+    let issueObservable: Observable<Issue>;
     if (this.issues === undefined) {
-      return this.getLatestIssue(id);
+      issueObservable = this.getLatestIssue(id);
     } else {
-      return of(this.issues[id]);
+      issueObservable = of(this.issues[id]);
     }
+    return issueObservable.pipe(
+      tap((issue) => {
+        if (issue.parseError) {
```

Comment:
Instead of assigning parseError to issue field, is it possible to throw an error in the observable and use rxjs in-built catchError to catch it? 

Catch and Rethrow: https://blog.angular-university.io/rxjs-error-handling/

Reply:
@snss231 Please check on @gycgabriel 's comment and update your PR so that we can proceed to merge!

Reply:
@gycgabriel @kkangs0226 The proposed solution no longer uses `tap`: `getIssue` is left untouched and all logic for handling parseErrors is in the `createIssueModel`. I think this is a simple solution to log all arcsecond errors as `createIssueModel` is essentially the only point in our application that works with the parser through the Issue factory methods. I explored throwing an issue in the `getIssue` observable, but this caused a lot of issues with the existing tests. Let me know what you think!

Reply:
![image](https://user-images.githubusercontent.com/77093722/191661178-8b33b6c7-d993-4b50-8143-c2528d61e954.png)

Reply:
![image](https://user-images.githubusercontent.com/77093722/191661983-33509ed1-4e2b-4580-96c5-7a22eb49ffa3.png)
Log message generated when the header is missing an 'e' ("Team Respons")

Reply:
I think the updated solution is better, looks good to me. @gycgabriel Let us know your thoughts

Reply:
Thanks for exploring throwing an error in the `getIssue` observable. I think that the updated solution is better because getIssue() is referenced by non-parser related functions as well (issue deletion by ID). We can proceed to merge soon if there are no other concerns.